iceoryx2-bb/elementary/src/package_version.rs

```diff
@@ -76,10 +76,17 @@ impl PackageVersion {
                 .and_then(|s| s.parse::<u16>().ok())
                 .unwrap_or(u16::MAX);
 
-            PACKAGE_VERSION.store(
-                PackageVersion::from_version(major, minor, patch).0,
-                Ordering::Relaxed,
-            );
+            if major == 0 && minor == 0 && patch == 0 {
+                PACKAGE_VERSION.store(
+                    PackageVersion::from_version(u16::MAX, u16::MAX, u16::MAX).0,
+                    Ordering::Relaxed,
```

Comment:
Would it make sense to just terminate when these variables are not set?

Also, is it possible that there are more env variables which are expected to be set by cargo and pitentially missing from bazel?

Reply:
I created an issue: #475 .

We need to refactor this and ensure that we do not depend on deployment env vars since they are obviously not always set as expected.

Reply:
Oh Wait ... I can fix it right here.